rockcraft.yaml

```
@@ -0,0 +1,71 @@
+name: spark
+summary: Spark ROCK
+description: Spark ROCK
+license: Apache-2.0
+
+version: "0.1"
+base: ubuntu:20.04
```

```suggestion
base: ubuntu:22.04
```

Probably worth basing on the latest LTS :)